### PR TITLE
Added some spacing when button has picture to look better

### PR DESCRIPTION
**Pull request by defnax**

### CODE REVIEW

**Comment by defnax** [comment]
@csoler @PhenomRetroShare

File: retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss [hunk 1]
```diff
@@ -2335,6 +2335,14 @@ QFrame#titleBarFrame QTextEdit {
 
 /**** Special Page tweak ****/
 
+/* Channels */
+
+CreateGxsChannelMsg QPushButton#channelpostButton, QPushButton#attachmentsButton,
+QPushButton#addThumbnailButton {
```
Comment (PhenomRetroShare) [inline retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: Prefer use generic names like btnWI_* (button With Image) so no need to add them one by one and get a big (messy) qss file.

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: oh you want that i change the names of the buttons? but i want only fix the issues without touch ui files.
normaly when some one touch to fix the qss styles he need touch edit the code too?

Comment (PhenomRetroShare) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: I don't want qss files becomes as messy as they were.
Here, you don't want to fix one thing but general one. So fix it in general way.

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: how to use generic names in qss? i dont understand that any example in qss how need to be look

like this?:
```
QFrame[objectName^="feedFrame"],/* Frame used in Feeds*/
QFrame[objectName^="plainFrame"],/* Frame used in Widget with plain background*/
QFrame[objectName^="plainBFrame"] {/* Frame used in Widget with plain bordered background*/
	background-color: #19232D;
}
```

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: but you doesnt fixed the buttons look and feel issues  in light & dark style

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: like this?:
QPushButton[objectName^="channelpostButton"] {
}

Comment (PhenomRetroShare) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: Rename buttons with image like "btnWI_ChannelPost"
In qss use:
QPushButton[objectName^="btnWI_"] {
...
}

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: ok mean rename first

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: im wondering why there are lots of issues on the buttons with spacing on dark/light style

image
image
image
image

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: "Cancel" button looks not normal too its a **QDialogButtonBox**

Comment (defnax) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: seems issues more with **QPushButton**, on **QToolButton**'s they have spacing with pictures

**Comment by defnax** [comment]
i has a fix for the QPushbutton's
https://github.com/RetroShare/RetroShare/pull/2525

File: retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss (hunk 1 above)
Comment (PhenomRetroShare) [reply retroshare-gui/src/gui/qss/stylesheet/Standard_Dark.qss:2341]: The fact that cancel button are smaller don't hurt me, on the contrary.

**Pull request closed without merging**